neuralop/layers/local_fno_block.py

```diff
@@ -17,108 +19,137 @@
+        Padding mode for spatial convolution kernels.
+    fin_diff_kernel_size : odd int, optional
+        Conv kernel size for finite difference convolution.
+    mix_derivatives : bool, optional
+        Whether to mix derivatives across channels.
+    max_n_modes : int, List[int], optional
+        maximum number of modes to keep along each dimension, by default None
+    fno_block_precision : str, optional
+        floating point precision to use for computations, by default "full"
+    channel_mlp_dropout : int, optional
+        dropout parameter for self.channel_mlp, by default 0
+    channel_mlp_expansion : float, optional
+        expansion parameter for self.channel_mlp, by default 0.5
+    non_linearity : torch.nn.F module, optional
+        nonlinear activation function to use between layers, by default F.gelu
+    stabilizer : Literal["tanh"], optional
+        stabilizing module to use between certain layers, by default None
+        if "tanh", use tanh
+    norm : Literal["ada_in", "group_norm", "instance_norm"], optional
+        Normalization layer to use, by default None
+    ada_in_features : int, optional
+        number of features for adaptive instance norm above, by default None
+    preactivation : bool, optional
+        whether to call forward pass with pre-activation, by default False
+        if True, call nonlinear activation and norm before Fourier convolution
+        if False, call activation and norms after Fourier convolutions
+    fno_skip : str, optional
+        module to use for FNO skip connections, by default "linear"
+        see layers.skip_connections for more details
+    channel_mlp_skip : str, optional
+        module to use for ChannelMLP skip connections, by default "soft-gating"
+        see layers.skip_connections for more details
+
+    Other Parameters
+    -------------------
+    complex_data : bool, optional
+        whether the FNO's data takes on complex values in space, by default False
+    separable : bool, optional
+        separable parameter for SpectralConv, by default False
+    factorization : str, optional
+        factorization parameter for SpectralConv, by default None
+    rank : float, optional
+        rank parameter for SpectralConv, by default 1.0
+    conv_module : BaseConv, optional
+        module to use for convolutions in FNO block, by default SpectralConv
+    joint_factorization : bool, optional
+        whether to factorize all spectralConv weights as one tensor, by default False
+    fixed_rank_modes : bool, optional
+        fixed_rank_modes parameter for SpectralConv, by default False
+    implementation : str, optional
+        implementation parameter for SpectralConv, by default "factorized"
+    decomposition_kwargs : _type_, optional
+        kwargs for tensor decomposition in SpectralConv, by default dict()
+    
+    References
+    -----------
+    .. [1] :
+
```

Comment:
The references here seem broken - I don't think this will get rendered properly

Reply:
This format for the reference I copied from `FNOBlocks`. @dhpitt Do you know if these would render correctly?

Reply:
There shouldn't be all the newlines: see e.g. https://github.com/tensorly/tensorly/blob/55a022e6558b281d63b2d342daf2a1cc2f2d8281/tensorly/decomposition/_cp.py#L316 for an example

Reply:
The references in FNOBlocks have the same issue.

Reply:
Ah, I see. I'll make a commit and fix both

Reply:
Just pushed!